app/questionnaire/routing/operations.py

```diff
@@ -49,6 +49,10 @@ def evaluate_or(values: Iterable[bool]) -> bool:
     return any(iter(values))
 
 
+def evaluate_count(values: Optional[Sized]) -> int:
+    return len(values) if values else 0
```

Comment:
We don't seem to exercise the else in any unit tests. This is strange as line coverage is 100%